internal/httputil/certlog.go

```diff
@@ -0,0 +1,150 @@
+package httputil
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/go-logr/logr"
+)
+
+const (
+	defaultLogLevel = 0 // 4
+)
+
+func DumpCertificates(path string, log logr.Logger) {
```

Comment:
Why we need to create this one?
Could we not use the cert generation of controller-runtime instead ?

Reply:
This displays (not generates) a specific selection of certificates.